_episodes/12-cluster.md

```diff
@@ -62,27 +62,14 @@ Go ahead and log in to the cluster: {{ site.remote.name }} at {{ site.remote.loc
 ```
 {: .bash}
 
-Remember to replace `yourUsername` with the username supplied by the instructors. You will be asked
-for your password. But watch out, the characters you type are not displayed on the screen.
+Remember to replace `yourUsername` with the username supplied by the instructors. You may be asked
+for your password. Watch out: the characters you type are not displayed on the screen.
 
 You are logging in using a program known as the secure shell or `ssh`. This establishes a temporary
 encrypted connection between your laptop and `{{ site.remote.login }}`. The word before the `@`
-symbol, e.g. `yourUsername` here, is the user account name that Lola has access permissions for on
-the cluster.
+symbol, e.g. `yourUsername` here, is the user account name that you permission to use on the
```

Comment:
There's a grammatical issue with "you permission to use..." here.

Reply:
f14c07b